### PR TITLE
Fadein effect on quiz and topic main page

### PR DESCRIPTION
**Pull request by arjunmehta94**
Fadein effect on the quiz and topic main pages.

### CODE REVIEW

File: app/components/annotation/Tua.js [hunk 1]
```diff
@@ -33,6 +35,7 @@ export default React.createClass({
     let topics = tua.analysis_type.topics;
 
     return (
+      <ReactCSSTransitionsGroup transitionName='fadein' transitionAppear>
```
Comment (mngyuan) [inline app/components/annotation/Tua.js:38]: nit: indent inside this tag

File: app/components/quiz/Quiz.js [hunk 2]
```diff
@@ -58,6 +60,7 @@ export default React.createClass({
   render() {
     var opts = this.state.answer.length !== 0 ? {} : {disabled: true};
     return (
+      <ReactCSSTransitionsGroup transitionName='fadein' transitionAppear>
```
Comment (mngyuan) [inline app/components/quiz/Quiz.js:63]: same as above indent

File: app/styles/Fadein.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.fadein-appear {
```
Comment (mngyuan) [inline app/styles/Fadein.scss]: nit: since it's not a stylesheet directly related to a react component, the filename should be `fadeIn.scss`

**Comment by mngyuan** [comment]
Sweet, fix those minor things and we'll be goin. Good stuff

**Comment by arjunmehta94** [comment]
Fixed.

**Comment by mngyuan** [comment]
d'oh - I think `react-css-transitions` should've ended up in `devDependencies` but it doesn't really matter

**Pull request merged**